Cargo.toml

```diff
@@ -13,22 +13,26 @@ rust-version = "1.60.0"
 
 # The async runtime features mirror those of `zbus` for compatibility.
 [features]
-default = ["async-io"]
+default = ["async-io", "rs-crypto"]
 async-io = ["zbus/async-io"]
 tokio = ["zbus/tokio"]
+rs-crypto = ["dep:aes", "dep:block-modes", "dep:sha2", "dep:hkdf"]
+vendored-openssl = ["openssl", "openssl/vendored"]
```

Comment:
Out of personal curiosity, is it possible to be FIPS compliant with `openssl/vendored`?

Reply:
Hah, good catch. It's not (at least not easily).

Quick note on semantics: FIPS compliant, certified, and certifiable are different things. Compliance generally (not a regulations expert, or a doctor) requires one side of the client/server to be certified. Though companies who work with governments often just require their entire supply chain to be certified rather than trying to nickle and dime each side...

My use case is an updated, Rust-based CLI for VS Code. At the moment, we'll vendor OpenSSL and therefore not be FIPS certifiable: this is okay for us since the product this is used in (VS Code) is also not FIPS certifiable, since we bundle Electron which in turn is not certifiable. However (1) there are separate standards within Microsoft that restrict the crypto libraries we can use, and (2) it is likely that at least the VS Code CLI (/some build of the CLI) _will_ be required to be FIPS certified in the future, if/when it's used in Azure offerings.